packages/@okta/vuepress-site/docs/release-notes/2021-okta-identity-engine/index.md

```diff
@@ -4,6 +4,61 @@ title: Okta Identity Engine API Products release notes 2021
+
+The Device Authorization grant feature is an OAuth 2.0 grant type that allows users to sign in to input-constrained devices and also to devices that lack web browsers. This feature enables users to use a secondary device, such as a laptop or mobile phone, to complete sign-in to applications that run on such devices. See [Configure Device Authorization](/docs/guides/device-authorization-grant/main/).<!--OKTA-450432-->
+
+#### Dynamic Issuer Mode is GA in Preview
+
+An authorization server's issuer URL can be used to validate whether tokens are issued by the correct authorization server. You can configure the issuer URL to be either the Okta subdomain (such as `company.okta.com`) or a custom domain (such as `sso.company.com`). See [Property details](/docs/reference/api/authorization-servers/#authorization-server-properties).
+
+When there are applications that use Okta's subdomain and other applications that use the custom domain, the issuer validation breaks because the value is hard-coded to one domain or the other.
+
+With Dynamic Issuer Mode, the issuer value in minted tokens is dynamically updated based on the URL that is used to initiate the original authorize request. See [Client application settings](/docs/reference/api/apps/#settings-10). <!--OKTA-447358-->
+
+#### Error response updated for malicious IP address sign-in requests
+
+If you block suspicious traffic and [ThreatInsight](/docs/reference/api/threat-insight/) detects that the sign-in request comes from a malicious IP address, Okta automatically denies the user access to the organization. The user receives an error in response to the request. From the user’s perspective, the blocked request can’t be identified as the result of ThreatInsight having identified the IP address as malicious. <!--OKTA-434409-->
+
+#### IdP Discovery supported for Device Authorization Grant flow
+
+The OAuth 2.0 [Device Authorization Grant flow](/docs/guides/device-authorization-grant/main/) now supports routing rules and IdP Discovery. <!--OKTA-425256-->
+
+#### Upload Logo for Org deprecated
+
+The Upload Logo for Org endpoint (`api/v1/org/logo`) is deprecated. Use the [Upload Theme Logo](/docs/reference/api/brands/#upload-theme-logo) (`/api/v1/brands/${brandId}/themes/${themeId}/logo`) endpoint instead. <!--OKTA-432207-->
+
+#### Bugs fixed in 2021.12.0
+
+* When the [Device Authorization Grant flow](/docs/guides/device-authorization-grant/main/) was used, token inline hooks weren't called. (OKTA-445422)
+
+* When users signed in to Okta Identity Engine using the [Sign-In Widget](/code/javascript/okta_sign-in_widget/), the **Username** field wasn't set as a required field. (OKTA-391311)
```

Comment:
```suggestion
* When users signed in to the Okta Identity Engine by using the [Sign-In Widget](/code/javascript/okta_sign-in_widget/), the **Username** field wasn't set as a required field. (OKTA-391311)
```

Reply:
@christinechoy-okta - when first introducing Okta Identity Engine - you don't use articles. On subsequent uses, it's THE Identity Engine
<img width="936" alt="Screen Shot 2021-12-07 at 4 32 39 PM" src="https://user-images.githubusercontent.com/45509385/145122203-58c24928-aa06-47ae-93c7-b0d9baf62629.png">
.

Reply:
👍